EDMarketConnector.py

```diff
@@ -123,6 +123,19 @@ def window_title(h):
 
                 @ctypes.WINFUNCTYPE(BOOL, HWND, LPARAM)
                 def enumwindowsproc(window_handle, l_param):
+                    """
+                    Determine if any window for the Application exists.
+
+                    Called for each found window by EnumWindows().
+
+                    When a match is found we check if we're being invoked as the
+                    edmc://auth handler.  If so we send the message to the existing
+                    process/window.  If not we'll raise that existing window to the
+                    foreground.
+                    :param window_handle: Window to check.
+                    :param l_param: ???
```

Comment:
```
Type: LPARAM

An application-defined value to be passed to the callback function.
```
https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-enumwindows
May as well note both of those